auto3dseg/algorithm_templates/dints/configs/hyper_parameters.yaml

```diff
@@ -49,10 +49,10 @@ training:
     momentum: 0.9
     weight_decay: 4.0e-05
   lr_scheduler:
-    _target_: torch.optim.lr_scheduler.StepLR
+    _target_: torch.optim.lr_scheduler.PolynomialLR
     optimizer: "$@training#optimizer"
-    step_size: "$max(@training#num_epochs // 5, 1)"
-    gamma: 0.5
+    power: 0.5,
+    total_iters: '$@training#num_epochs // @training#num_per_validation_epochs + 1'
```

Comment:
Thank you for the PR @dongyang0122 .

`num_per_validation_epochs` seems unreferenced to me.  Do you mean `num_epochs_per_validation`?

Reply:
current parser cannot understand `num_epochs_per_validation` if we have already defined `num_epochs` in the .json config.